TPO/src/main/java/org/uade/adt/Set.java

```diff
@@ -13,15 +14,23 @@ public Set() {
     }
 
     @Override
-    public void add(int a) {
+    public ISet Add(int a) {
         for(int i = 0; i < this.count; i++) {
             if(this.array[i] == a) {
-                return;
+                return null;
```

Comment:
Tiene que hacer return this también. Sino, al hacer .add de un valor existente, y a eso hacerle otro .add, va a tirar un NPE (NullPointerException)

Reply:
Actualizado